trestle/core/commands/author/profile.py

```diff
@@ -77,17 +105,20 @@ def _run(self, args: argparse.Namespace) -> int:
                     logging.warning(f'YAML error loading yaml header for ssp generation: {e}')
                     return CmdReturnCodes.COMMAND_ERROR.value
 
+            # combine command line sections with any in the yaml header, with priority to command line
+            sections_dict: Optional[Dict[str, str]] = None
+            if 'sections' in args and args.sections:
```

Comment:
I believe the first check is not necessary

Reply:
will be handled later by issue #1031